apps/web/src/lang/fr.json

```diff
@@ -2151,8 +2151,8 @@
     "defaultMessage": "Ajouter une nouvelle expérience",
     "description": "A link to add a new experience to your career timeline"
   },
-  "ARL2Tz": {
-    "defaultMessage": "Au moyen de la liste des compétences fournie dans l’annonce du bassin de candidats, sélectionnez les compétences que vous prévoyez évaluer au moyen de cette méthode d’évaluation.",
+  "ylpxzO": {
+    "defaultMessage": "Au moyen de la liste des compétences du processus de recrutement, sélectionnez les compétences que vous prévoyez évaluer au moyen de cette méthode d’évaluation.",
```

Comment:
Is a recruitment process different than a hiring process?

https://github.com/GCTC-NTGC/gc-digital-talent/blob/3d2d51b3cd328564e7707e0e4879939b85341331/api/config/rolepermission.php#L536-L539

Reply:
Hiring process and recruitment process are the same thing